components/service/pocket/src/main/java/mozilla/components/service/pocket/GlobalDependencyProvider.kt

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.pocket
+
+import android.annotation.SuppressLint
+import mozilla.components.service.pocket.spocs.SpocsUseCases
+import mozilla.components.service.pocket.stories.PocketStoriesUseCases
+
+/**
+ * Provides global access to the dependencies needed for updating Pocket stories.
+ */
+internal object GlobalDependencyProvider {
+    internal object RecommendedStories {
```

Comment:
Created different `RecommendedStories` and `SponsoredStories` objects because I saw both features - recommended stories and sponsored stories might have the same dependencies - like the http client.
And one feature can be disabled - as such we'll cleanup it's dependencies
While the other one will still need to be able to refresh it's data.
Separating the dependencies like here seemed like the simplest way to allow for that.